modules/howtos/examples/logging_example.py

```diff
@@ -0,0 +1,19 @@
+# tag::logging[]
+import logging
+import sys
+
+import couchbase
+from couchbase.auth import PasswordAuthenticator
+from couchbase.cluster import Cluster
+from couchbase.options import ClusterOptions
+
+logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
+couchbase.enable_logging()
```

Comment:
@thejcfactor or @davidkelly, just wanted to double check, has this changed in SDK 4.0?
I couldn't find a way to get `couchbase.enable_logging()` to work, which made me think that it's perhaps not working yet or will change?

Reply:
Sorry for the delay!  You are correct, couchbase.enable_logging() is not in the 4.x SDK.  I do not know if we will implement that or not in a future release.  I believe it will be a decision we can make once we have the log forwarding available within couchbase++.